qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -1,20 +1,19 @@
-- qemu_guest_agent: install setup image_copy unattended_install.cdrom
+- qemu_guest_agent: install setup image_copy unattended_install.cdrom single_driver_install.with_vioserial
```

Comment:
This won't run automatically , Just a tip showing that those guest agent cases will depend on that cases in test loop